tailwind.config.js

```diff
@@ -67,6 +68,24 @@ module.exports = {
           "0%": { transform: "translateX(100%)" },
           "100%": { transform: "translateX(0)" },
         },
+        fadeIn: {
+          "0%": { opacity: "0", transform: "translateY(-1px)" },
+          "95%": { opacity: "1", transform: "translateY(-1px)" },
+          "100%": { opacity: "1", transform: "translateY(0px)" },
+        },
+        fadeIn85: {
+          "0%": { opacity: "0", transform: "translateY(-1px)" },
+          "95%": { opacity: "0.85", transform: "translateY(-1px)" },
```

Comment:
I dont get it. Why need to translate the Y point with -1 pixel?

Reply:
Maybe I understand it incorrectly: when I play the "Flow 2" in Figma, there seems a slight drop for the position of the title after fade-in. If it is not the designated purpose I will just remove this Y point translate.